src/SyncFrom/Manager.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace Drupal\entity_sync\SyncFrom;
```

Comment:
@shabananavas While I liked the From and To we used in the HubSpot module, it's simple, I think it's best to use Import and Export because it's clearer. `From` by itself doesn't imply a direction, from Drupal or from the remote? While being within Drupal saying Import is clear that is an import from the remote into Drupal.

In the Sync admin section we will also use that terminology e.g. `Sync/Companies/Import` etc.

So let's make this `entity_sync/src/Import/Manager.php`.